waspc/src/Wasp/Analyzer/Parser/Parser.y

```diff
@@ -67,7 +67,7 @@ Stmts :: { AST }
   | Stmts StmtWithCtx { AST $ astStmts $1 ++ [$2] }
 
 StmtWithCtx :: { WithCtx Stmt }
-  : ctx Stmt { WithCtx $1 $2 }
+  : posStart Stmt posEnd { WithCtx (ctxFromRgn $1 $3) $2 }
```

Comment:
Now we have these two productions, posStart and posEnd, that capture the region!